### ISSUE TITLE
Build Failed in Android Platfrom

### ISSUE THREAD

**Issue by ghost** [zino-hofmann/graphql-flutter#1466]
**Describe the issue**
After selecting the platform as Android, an error occurs when building the Flutter project.
This issue seems to be related to the version of the ’connectivity_plus‘ package.

**To Reproduce (MUST BE PROVIDED)**

1. create a new empty flutter project 
2. run `flutter pub add graphql_flutter`
3. select AVD(android vitual device)
4. run main.dart
5. build failed

**device / execution context**

- flutter version:3.24.5
- android api 34
- graphql_flutter 5.1.2

**screenshots**

```
Waiting for another flutter command to release the startup lock...
Launching lib/main.dart on sdk gphone64 arm64 in debug mode...

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':connectivity_plus'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Specify a namespace in the module's build file. See https://d.android.com/r/tools/upgrade-assistant/set-namespace for information about setting the namespace.

     If you've specified the package attribute in the source AndroidManifest.xml, you can use the AGP Upgrade Assistant to migrate to the namespace value in the build file. Refer to https://d.android.com/r/tools/upgrade-assistant/agp-upgrade-assistant for general information about using the AGP Upgrade Assistant.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 21s
Error: Gradle task assembleDebug failed with exit code 1

Exited (1).

```

**Comment by ufumerfarooq67** [comment]
Hi,

I am also getting the same error with this package however the simple graphql runs fine.

**Comment by vincenzopalazzo** [comment]
Did you try to use the beta version? it should be stable and probably will be released in December

**Comment by ashen-n** [comment]
It's my another account, I didn't try beta version, I just use flutter pub add. Got it, I will try it later.

**Comment by vincenzopalazzo** [comment]
> It's my another account, I didn't try beta version, I just use flutter pub add. Got it, I will try it later.

yes please try to work always with the beta one that it is the more recent one and it is not unstable, the alpha is consider unstable

**Comment by henriqueczu** [comment]
Same issue

```
* What went wrong:
A problem occurred configuring project ':connectivity_plus'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Specify a namespace in the module's build file. See https://d.android.com/r/tools/upgrade-assistant/set-namespace for information about setting the namespace.

```

**Comment by guns28** [comment]
same here 

* What went wrong:
A problem occurred configuring project ':connectivity_plus'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Specify a namespace in the module's build file. See https://d.android.com/r/tools/upgrade-assistant/set-namespace for information about setting the namespace.

**Comment by AhmedAlaaGenina** [comment]
any update

**Comment by henriqueczu** [comment]
> any update
> 

not yet

**Comment by guns28** [comment]
for me i just forked it and updated connectivity_plus version and it worked. this is related to the new android configuration

**Comment by vincenzopalazzo** [comment]
>for me i just forked it and updated connectivity_plus version and it worked. this is related to the new android configuration

what version are you on?

Why did you not open a PR to fix it?

**Comment by khanderz** [comment]
> > It's my another account, I didn't try beta version, I just use flutter pub add. Got it, I will try it later.
> 
> yes please try to work always with the beta one that it is the more recent one and it is not unstable, the alpha is consider unstable

just tried with the beta version and I am still getting the same error. someone recently suggested to just fork it and updated connectivity_plus, but I dont want to get locked into that forked repo
